sklearn/utils/validation.py

```diff
@@ -1369,6 +1369,44 @@ def check_symmetric(array, *, tol=1e-10, raise_warning=True, raise_exception=Fal
     return array
 
 
+def _is_fitted(estimator, attributes=None, all_or_any=all):
```

Comment:
A couple of unit tests to ensure the behaviour :)
We could at least have the same test than `check_is_fitted`: when an error is raised -> False, otherwise True.

Reply:
I reused an existing test to check consistency between the 2.